myVK/myVK/ViewController/Controller/Collection/FriendPhotoCollectionViewController.swift

```diff
@@ -47,6 +59,8 @@ extension FriendPhotoCollectionViewController {
             ) as? FriendPhotosCollectionViewCell
         else { return UICollectionViewCell() }
         cell.updateCell(imageName: userPhotoNames[indexPath.row])
+        cell.photoNames = userPhotoNames
```

Comment:
```suggestion
        cell.photoNames = userPhotoNames
        cell.currentPhotoIndex = indexPath.row
```
передавай тоже через конфигур